README.md

```diff
@@ -163,9 +163,24 @@ require 'active_record_shards/sql_comments'
 ActiveRecordShards::SqlComments.enable
 ```
 
+## Changelog
+
+We use [github-changelog-generator](https://github.com/github-changelog-generator/github-changelog-generator) gem to keep our changelog updated.
+
+```Shell
+bundle exec github_changelog_generator \
+  --user zendesk \
+  --project active_record_shards
```

Comment:
these should be hardcoded in a rake task ?
`TOKEN=your-token-here rake update_changelog`

Reply:
Fixed!